website/docs/metadata.md

```diff
@@ -61,6 +62,11 @@ Following are the different types of metadata currently supported.
     hoodie.metadata.index.column.stats.columns=<comma-separated-column-names>
   ```
 
+- ***Secondary Index***
```

Comment:
@codope @vc I have added Secondary Index here. I see that Expression index, record index, bloom filter index is missing on this page but are covered in `website/docs/indexes.md`.
Should we rather refer to indexes page in this doc?

Reply:
I want this page to be about metadata (col stats, file listings, partition stats). lets do indexes on the indexes page

Reply:
Reverted this change.